particle-accelerator/.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
This husky thing looks pretty cool. Did you find it useful? Did it prevent you from committing incorrectly formatted code? You might need to turn this off to show the linter failing on Travis as I suggested, but if you like it then it might be something that I'll need to add to the rest of the code bases.

Reply:
This was another framework from the boilerplate. I didn't actually use it, and after removing the .git from the particle-accelerator directory, I've removed it from the PA directory as it no longer functions.